src/Greenshot.Base/Core/CoreConfiguration.cs

```diff
@@ -496,6 +499,17 @@ public override void BeforeSave()
         /// </summary>
         public override void AfterLoad()
         {
+            // Migrate old OutputFileCopyPathToClipboard boolean to new PostSaveBehavior enum
+            // Once migrated, the old property is set to null and won't exist in the INI file anymore
+            if (OutputFileCopyPathToClipboard.HasValue)
+            {
+                OutputFilePostSaveBehavior = OutputFileCopyPathToClipboard.Value
+                    ? PostSaveBehavior.CopyFilePathToClipboard
+                    : PostSaveBehavior.None;
+                // Set to null so it won't be written back to the INI file (ExcludeIfNull)
+                OutputFileCopyPathToClipboard = null;
```

Comment:
```suggestion
                OutputFileCopyPathToClipboard = null;
                IsDirty = true;
```

To force a early saving after reading.